lib/ansible/modules/cloud/azure/azure_rm_mysqlserver.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_mysqlserver
+version_added: "2.5"
+short_description: Manage MySQL Server instance.
+description:
+    - Create, update and delete instance of MySQL Server.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    name:
+        description:
+            - The name of the server.
+        required: True
+    sku:
+        description:
+            - The SKU (pricing tier) of the server.
+        suboptions:
+            name:
+                description:
+                    - The name of the sku, typically, a letter + Number code, e.g. P3.
+            tier:
+                description:
+                    - The tier of the particular SKU, e.g. Basic.
+                choices: ['basic', 'standard']
+            capacity:
+                description:
+                    - "The scale up/out capacity, representing server's compute units."
+            size:
+                description:
+                    - The size code, to be interpreted by resource as appropriate.
+    location:
+        description:
+            - Resource location. If not set, location from the resource group will be used as default.
+    storage_mb:
+        description:
+            - The maximum storage allowed for a server.
+    version:
+        description:
+            - Server version.
+        choices: ['5.6', '5.7']
+    enforce_ssl:
+        description:
+            - Enable SSL enforcement.
```

Comment:
Because this is a boolean value you should add the following
```
enforce_ssl:
  description:
  - Enable SSL enforment.
  type: bool
  default: 'no'
```

The `type: bool` means the docs will automatically put `yes` and `no` for the choices on the generated docs, the default should be quoted and can be `yes` or `no` depending on what the actual default it.

Reply:
added